builder/dockerfile/internals.go

```diff
@@ -137,6 +159,70 @@ func (b *Builder) performCopy(state *dispatchState, inst copyInstruction) error
 	return b.exportImage(state, imageMount, runConfigWithCommentCmd)
 }
 
+func parseChownFlag(chown, ctrRootPath string, idMappings *idtools.IDMappings) (idtools.IDPair, error) {
+	var userStr, grpStr string
+	parts := strings.Split(chown, ":")
+	if len(parts) == 1 {
+		// if no group specified, use the user spec as group as well
+		userStr, grpStr = parts[0], parts[0]
+	}
+	userStr, grpStr = parts[0], parts[1]
```

Comment:
This line will will panic when `len(parts) == 1` , it's not in an else.

This function should have unit tests.